hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java

```diff
@@ -417,4 +425,86 @@ private List<ContainerBlockMetadata> getBlocks(
   private BucketLayout getBucketLayout() {
     return BucketLayout.DEFAULT;
   }
+
+  @GET
+  @Path("/mismatch")
+  public Response getContainerMisMatchInsights() {
+    List<ContainerDiscrepancyInfo> containerDiscrepancyInfoList =
+        new ArrayList<>();
+    try {
+      Map<Long, ContainerMetadata> omContainers =
+          reconContainerMetadataManager.getContainers(-1, -1);
+      List<Long> scmAllContainers = containerManager.getContainers().stream()
+          .filter(containerInfo -> !(containerInfo.getState() ==
+              HddsProtos.LifeCycleState.DELETED))
+          .map(containerInfo -> containerInfo.getContainerID()).collect(
+              Collectors.toList());
+      List<Long> scmNonDeletedContainers =
```

Comment:
scmNonDeletedContainers and scmAllContainers are same, duplicate

Reply:
> scmNonDeletedContainers and scmAllContainers are same, duplicate

Its fixed. Pls re-review.